src/toast/scripts/toast_benchmark_ground

```diff
@@ -43,6 +52,13 @@ def parse_arguments():
     parser = argparse.ArgumentParser(
         description="Run a TOAST ground workflow scaled appropriately to the MPI communicator size and available memory."
     )
+    parser.add_argument(
```

Comment:
We might want to make that argument mandatory rather than optional. If a user runs the script blindly they might be thrown off by the default folder name not existing and be unclear on what to do.

Also the help might mention the script used to generate the inputs.